app/components/editor/layouts/Classic.tsx

```diff
@@ -0,0 +1,56 @@
+import cx from 'classnames';
+import BaseLayout, { LayoutProps } from './BaseLayout';
+import { createProps } from 'components/tsx-component';
+import { Component } from 'vue-property-decorator';
+import ResizeBar from 'components/shared/ResizeBar.vue';
+import styles from './Layouts.m.less';
+
+@Component({ props: createProps(LayoutProps) })
+export default class Classic extends BaseLayout {
+  mounted() {
+    super.mountResize();
```

Comment:
Why not `this.mountResize`?  Seems incorrect to use super in this case, since you should allow `mountResize` to potentially be overridden.  (This comment applies to all layout implementations)